container-images/rocm-fedora/Containerfile

```diff
@@ -1,8 +1,10 @@
 FROM registry.fedoraproject.org/fedora:latest
 
+RUN dnf update -y --refresh && dnf clean all
```

Comment:
This step doesn't seem unique to this Containerfile or anything, no big deal though